### PR TITLE
(#966) - Document replication retry option

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -616,8 +617,6 @@ var sync = PouchDB.sync('mydb', 'http://localhost:5984/mydb', {live: true})
     // handle change
   }).on('complete', function (info) {
     // handle complete
-  }).on('uptodate', function (info) {
-    // handle up-to-date
   }).on('error', function (err) {
```
Comment (nolanlawson) [inline docs/api.md]: shouldn't we document `'paused'` down here?

Comment (daleharvey) [inline docs/api.md]: This wasnt an exhaustive list so I didnt thought that was a little confusing, change / complete and error are the primary events. I want to have a more explanatory documentation but that needs a little prework to make sure it aligns with the guide / api docs

Comment (nolanlawson) [inline docs/api.md]: OK, in that case +1, but yeah, we need more docs.

Probably also a big blog post introducing the new feature, and also a new section in the guide.

Comment (daleharvey) [inline docs/api.md]: Yeh the next on my list after your android commit is a docs walkthrough

**Comment by daleharvey** [comment]
Sweet - https://github.com/pouchdb/pouchdb/commit/ecc03209bea7e603ec8cf2d241be207ffa2c0cb8

**Pull request closed without merging**